src/main/java/net/javadiscord/javabot/api/routes/metrics/MetricsController.java

```diff
@@ -42,22 +47,23 @@ public MetricsController() {
 	 * @return The {@link ResponseEntity}.
 	 */
 	@GetMapping(
-			value = "{guild_id}/metrics",
-			produces = MediaType.APPLICATION_JSON_VALUE
+			value = "{guild_id}/metrics"
```

Comment:
[`value=` is unnecessary if no other parameters are given to an annotation.](https://docs.oracle.com/javase/specs/jls/se18/html/jls-9.html#jls-SingleElementAnnotation)